front/src/styles/components/Toasts/Toast.styles.ts

```diff
@@ -0,0 +1,44 @@
+import { css } from '@emotion/react'
+import { theme } from '@/styles/theme'
+
+export const styles = {
+  toastWrapper: (type: string, visible: boolean) => css`
+    position: fixed;
+    top: 0px;
```

Comment:
```suggestion
    top: 0;
    left: 0;
```

leftも指定しないと、端っこはみ出ちゃったりする場合あります！
↓Storybookの右端
<img width="1224" alt="スクリーンショット 2023-07-04 1 22 17" src="https://github.com/traveli-dev/triplate-app/assets/74092547/0cb9f694-ba45-42d5-a633-af8dfe045a42">